docs/develop-resources/guides/5-releases.md

```diff
@@ -1,14 +1,10 @@
 # 5. Release
 
-## 5.1 Turn on your Direct controller (TF/DCL Beta Only)
+## 5.1 Make Direct controller the default (for migration from TF/DCL)
```

Comment:
You may want to keep the `Beta only` because Alpha TF/DCL migration does not require changing the tf2crd or dcl2crd labels.